test/parallel/test-fs-read-offset-null.js

```diff
@@ -31,12 +40,16 @@ let filehandle = null;
 // Test for promise api
 (async () => {
   filehandle = await fsPromises.open(filepath, 'r');
-  const readObject = await filehandle.read(buf, null, buf.length);
```

Comment:
Don't we want to keep this test as is? If it's not semver-major, all tests should still work.

Reply:
Agreed, keeping both versions of this test.
In the original version, `position` as positional argument is omitted. This is implicitly allowed by the code and there's no problem to allow it in the docs, if needed.

Reply:
Ugh, actually there should be explicit bigint support, and this variable should meet `validatePosition()` so it won't be literally anything anymore. `null` should be exceptionally allowed, `undefined` could be prohibited.
Anyways, in a context of this PR both tests are still applicable.